root/types.js

```diff
@@ -687,7 +688,7 @@ declare type LinkTypeT = {
   +attributes: {+[typeId: number]: LinkTypeAttrTypeT},
   +cardinality0: number,
   +cardinality1: number,
-  +children?: $ReadOnlyArray<LinkTypeT>,
+  +children?: Array<LinkTypeT>,
```

Comment:
I think this and above should stay `$ReadOnlyArray` (if sorting is the issue, I'd prefer doing that on the server if possible, as already mentioned.)